frontend/src/components/Search/SearchContext.tsx

```diff
@@ -104,6 +114,39 @@ export const SearchContext: React.FC<Props> = ({
 	const { queryParts, tokens } = parseSearch(query)
 	const tokenGroups = buildTokenGroups(tokens)
 
+	const [pendingStartDate, setPendingStartDate] = useState<Date>()
+	const [pendingEndDate, setPendingEndDate] = useState<Date>()
+	const [pendingSelectedPreset, setPendingSelectedPreset] =
+		useState<DateRangePreset>()
```

Comment:
Copy of date options to allow editing without submitting search

Reply:
Do we feel strongly about the UX of allowing editing the query before submitting? Feels like we could potentially simplify things in the code + UI by just submitting it 🤔 

cc @jay-khatri @julian-highlight

Reply:
Don't feel too strongly. If it simplifies things, let's remove the edit option! Wdyt @SpennyNDaJets?

Reply:
Sure thing!